radixdlt/src/main/java/com/radixdlt/consensus/validators/Validator.java

```diff
@@ -80,56 +64,32 @@
 	private final boolean jailed;
 */
 
-	@JsonProperty("power")
```

Comment:
I think it's a mistake to remove the serialisation stuff here, as we almost certainly will want to persist these at some point.

Reply:
Even when we do persist these, I would rather have Data object be different from Functional object. Even so, since we're not serializing these yet I think premature to already decide on this class being the serialization mechanism.